examples/tensorflow_eager_simple.py

```diff
@@ -51,7 +54,7 @@ def create_model(trial):
 def create_optimizer(trial):
     # We optimize the choice of optimizers as well as their parameters.
     kwargs = {}
-    optimizer_options = ['RMSPropOptimizer', 'AdamOptimizer', 'MomentumOptimizer']
+    optimizer_options = ['RMSprop', 'Adam', 'SGD']
     optimizer_selected = trial.suggest_categorical('optimizer', optimizer_options)
     if optimizer_selected == 'RMSPropOptimizer':
```

Comment:
Hi @nuka137 ! I'm not familiar with tensorflow, but it looks these conditions to set optimizer's arguments should be updated like:

```python
if optimizer_selected == 'RMSProp':
    kwargs['learning_rate'] = trial.suggest_loguniform('rmsprop_learning_rate', 1e-5, 1e-1)
    ...
elif optimizer_selected == 'Adam':
    ...
elif optimizer_selected == 'SGD':
    ...
```

Reply:
@c-bata 
Good catch! Thank you.

Reply:
@c-bata 

Thanks! I forgot to fix these.

@toshihikoyanase 

I also fixed the parameter names for `suggest_loguniform`.
Could you check them, please?